src/ostree/ot-builtin-summary.c

```diff
@@ -40,6 +40,63 @@ static GOptionEntry options[] = {
   { NULL }
 };
 
+/* Take arguments of the form [key=value] or [key, value] out of @argv and put
+ * them into an a{sv} variant. The value arguments must be parsable using
+ * g_variant_parse(). */
```

Comment:
Hmm.  A bit ad-hoc.  If we're taking `g_variant_parse()`, is there a reason not to just require the user to pass the whole `a{sv}` that way?  Presumably this will be used in scripts where that'll be...well, scripted and not impact the interactive experience.

We have some prior art here for `ostree commit --add-metadata-string`; do we expect non-string keys here?